src/main/utils/geometry/boundary.ts

```diff
@@ -47,3 +47,11 @@ export function computeBoundingBoxForElements(elements: { bounds: IBoundary }[])
   const height = Math.max(...boundaries.map((bounds) => bounds.y + bounds.height)) - y;
   return { x, y, width, height };
 }
+
+export function computeDimension(scale: number, value: number, isCircle?: boolean): number {
```

Comment:
Could you maybe use the scale factor from `getScaleFactor` function (defined [here](https://github.com/ls1intum/Apollon/blob/05ab7674add6700bff26939a7299d9308fa3b9e1/src/main/apollon-editor.ts#L282)), instead of using it as parameter?

Reply:
Thanks for the suggestion @minrows, it makes sense, but we don't have access to editor.getScaleFactor, and using editor from the state would add more complexity since this is not a react function/component.